packages/docs/content/guide/kr/02. customization-guide/01. structure-and-customization.md

```diff
@@ -0,0 +1,175 @@
+              }
+            ],
+            "color": "default"
+          }
+        }
+      ]
+    }
+  ]
+}
+```
+
+### 주요 구조 요소
+
+1. **페이지 메타데이터**: `id`, `created_time`, `last_edited_time`, `parent` 등 페이지의 기본 정보를 포함합니다.
+
+2. **cover와 icon**: 페이지의 커버 이미지와 아이콘 정보입니다. 커스텀 디자인을 적용할 때 이 정보를 활용할 수 있습니다.
+
+3. **properties**: 페이지의 속성 정보로, 주로 페이지 제목(`title`)이 포함됩니다. 커스텀 헤더를 만들 때 이 정보를 사용할 수 있습니다.
+
+4. **blocks**: 페이지 내용을 구성하는 블록들의 배열입니다. 각 블록은 다음 구조를 가집니다:
+
+   - `object`: 항상 "block"
+   - `id`: 블록의 고유 식별자
+   - `type`: 블록의 타입 (예: "toggle", "paragraph", "heading_1" 등)
+   - `[type]`: 블록 타입에 해당하는 키로, 해당 블록의 구체적인 내용을 포함합니다.
+   - `blocks`: 하위 블록들의 배열. 이를 통해 재귀적인 구조를 형성합니다.
+
+5. **재귀적 구조**: `blocks` 배열 내의 각 블록은 다시 `blocks` 배열을 가질 수 있어, 중첩된 구조를 표현할 수 있습니다. 이는 토글, 목록 등의 중첩된 콘텐츠를 커스터마이징할 때 중요합니다.
+
+6. **rich_text**: 대부분의 텍스트 콘텐츠는 `rich_text` 배열로 표현되며, 텍스트의 스타일 정보(`annotations`)를 포함합니다. 이를 통해 텍스트 스타일을 세밀하게 커스터마이징할 수 있습니다.
+
+## 커스터마이징을 위한 컴포넌트 구조
+
+React-Notion-Custom의 기본적인 컴포넌트 구조는 다음과 같습니다:
+
+```
+┌────────────────────────────────────────────────────────────────┐
+│                           <Notion>                             │
+│  ┌────────────────────────────────────────────────────────┐    │
+│  │                        <Cover />                       │    │
+│  │                  // 페이지 커버 이미지                      │    │
+│  └────────────────────────────────────────────────────────┘    │
+│                                                                │
+│               ┌────────────────────────────┐                   │
+│               │           <Body>           │                   │
+│               │  ┌──────────────────────┐  │                   │
+│               │  │      <Title />       │  │                   │
+│               │  │    // 페이지 제목       │  │                   │
+│               │  └──────────────────────┘  │                   │
+│               │  ┌──────────────────────┐  │                   │
+│               │  │      <Blocks>        │  │                   │
+│               │  │ ┌──────────────────┐ │  │                   │
+│               │  │ │<Block type="..." │ │  │                   │
+│               │  │ │   // 예: 헤딩      │ │  │                   │
+│               │  │ └──────────────────┘ │  │                   │
+│               │  │ ┌──────────────────┐ │  │                   │
+│               │  │ │<Block type="..." │ │  │                   │
+│               │  │ │   // 예: 단락    │ │  │                   │
+│               │  │ └──────────────────┘ │  │                   │
+│               │  │         ...          │  │                   │
+│               │  │  // 추가 블록들      │  │                   │
+│               │  └──────────────────────┘  │                   │
+│               └────────────────────────────┘                   │
+│                                                                │
+└────────────────────────────────────────────────────────────────┘
```

Comment:
p3: It looks like the current markdown structure is slightly breaking. It might need some adjustments to ensure everything is displayed correctly.

Reply:
wow.. I will fix it